src/nodes/comment.gd

```diff
@@ -0,0 +1,33 @@
+tool
+extends ConceptNode
+
+
+func _init() -> void:
+	unique_id = "comment"
+	display_name = "Comment"
+	category = "Utilities"
+	description = "Insert comment in concept graph editor"
+	resizable = true
+
+func export_custom_data() -> Dictionary:
+	return {"comment_text": $LineEditComment.text}
+
+func restore_custom_data(data: Dictionary) -> void:
+	if not data.has("comment_text"):
+		return
+	$LineEditComment.text = data["comment_text"]
+
+func restore_editor_data(data: Dictionary) -> void:
+	.restore_editor_data(data)
+	# we need to emit resize request signal to restore rect size
+	if data.has("rect_x") && data.has("rect_y"):
+		emit_signal("resize_request", Vector2(data["rect_x"], data["rect_y"]))
```

Comment:
I pushed a fix for the graph edit ignoring the node size, you made me realize I had a typo there too.
Try it with the latest commit on master, this whole function should not be needed anymore

Reply:
Ok it fixed it.